### ISSUE TITLE
Conferencing minutes change from 60 to 900?

### ISSUE THREAD

**Issue by Bmumpower2929** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#8408]
Has the complimentary dialout period been increased from 60 minutes to 900 minutes?
https://docs.microsoft.com/en-us/microsoftteams/complimentary-dial-out-period

If so then the following on this page should be updated to reflect this change.
No. The existing Audio Conferencing 60 minutes per user per month, pooled at the tenant level, will continue to apply to all Zone A dial-out conferencing countries.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b4345161-db6e-1cf8-59fe-4330b4cc2852
* Version Independent ID: c5f811f1-9501-2ab6-8dbb-2868ccf90c1c
* Content: [Office 365 Audio Conferencing Extended Dial-out Minutes to US and Canada - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/dial-out-minutes-canada-us)
* Content Source: [Teams/dial-out-minutes-canada-us.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/dial-out-minutes-canada-us.md)
* Service: **msteams**
* GitHub Login: @MicrosoftHeidi
* Microsoft Alias: **heidip**

**Comment by Reeced40** [comment]
@Bmumpower2929 thank you for reaching out. As far as I know this complimentary dial-out period ended in December 2019, so the article will stay as is. Thanks again for your help in improving Microsoft Docs.

**Comment by Bmumpower2929** [comment]
Then when did it go from 60 minutes to 900 minutes? It use to be such a small amount we had to turn it off cause it was laughable. But 900 minutes might actually be usable.  But now with that update there is contradictory because they have unlimited for certain skus.

The articles were poorly updated.

**Comment by Bmumpower2929** [comment]
The article still doesn't explain why the minutes went from 60-900 minutes. If i understand correctly the 900 minutes are only going to be needed for out of country. The article just says it's unlimited and doesn't say any exclusions. 

I think linking to  this article and fixing the wording would make more sense.
https://docs.microsoft.com/en-us/microsoftteams/complimentary-dial-out-period
 
But then looking at the change history of that article looks like someone said something about us not getting 900 minute 2 years ago. and they referenced this article.
https://docs.microsoft.com/en-us/microsoftteams/audio-conferencing-subscription-dial-out#how-are-minute-pools-calculated


TL;DR these articles are a wild goose chase

**Comment by Reeced40** [comment]
@Bmumpower2929 unfortunately the benefit ended in 2019 so it was only for a certain period of time that they had 900 minutes. The article you referenced above is linked to this document at the bottom of the page to help other users. Thanks again for reaching out.

**Comment by Reeced40** [comment]
@scanum this can be closed thank you.
